app/controllers/settings/index.js

```diff
@@ -120,6 +120,10 @@ function openAbout() {
     nav.openWindow(Alloy.createController("/settings/webview", "about").getView());
 }
 
+function openContributors(e){
+  nav.openWindow(Alloy.createController("/lunches/details/contributors", "contributors").getView());
```

Comment:
Please use 4-space indentation and fix it for Android (see other classes for common `open` examples)